### PR TITLE
Add Carrd

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Carlgo11** [comment]
Thanks for the pull request!

I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

**Comment by phallobst** [comment]
FYI: https://carrd.co/docs/account/enabling-google-authenticator

File: _data/other.yml [hunk 1]
```diff
@@ -69,6 +69,12 @@ websites:
     twitter: CAinc
     facebook: CATechnologies
 
+  - name: Carrd
+    url: https://carrd.co/
+    img: carrd.png
+    tfa:
+      - totp
+
```
Comment (Carlgo11) [inline _data/other.yml:78]:
```suggestion
    doc: https://carrd.co/docs/account/enabling-google-authenticator

```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Please move the site to the `Hosting/VPS` category and update your branch to our repo to fix the merge conflicts.

**Comment by Carlgo11** [comment]
As I haven't seen any activity from @nyuszika7h since the creation of the PR, I'll close this PR after a week unless a response is made within that time window.

**Pull request closed without merging**